### PR TITLE
Cg refactor 2

### PR DESCRIPTION
**Pull request by cgaddis36**
Fixed admin dashboard to contain a link to each orders show page shown on each orders id

-Added in tests for item show page of merchant to test for the css of the item

-Made the item appear in the correct show

-Added in status_changer method to the fulfill method to ensure the order's status is changed when the last item is fulfilled in an order

-Fixed order show page route and created admin order view page as well as the show method in the admin/orders_controller

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -56,6 +56,7 @@
     get '/', to: 'dashboard#index'
     get '/users', to: 'users#index'
     get '/users/:user_id', to: 'users#show'
+    get '/users/:user_id/orders/:order_id', to: 'orders#show'
```
Comment (adumortier) [inline config/routes.rb:59]: the restful way, I like this route

File: db/seeds.rb [hunk 2]
```diff
@@ -9,47 +9,6 @@
 Merchant.destroy_all
 Item.destroy_all
 
-# Users
```
Comment (adumortier) [inline db/seeds.rb:12]: thanks for fixing the seeds

File: spec/features/users/merchant_order_show_spec.rb [hunk 3]
```diff
@@ -108,7 +108,7 @@
 
     within "#item-#{@item_1.id}" do
       expect(page).to have_content(@item_1.name)
-      expect(page).to have_content(@item_1.image)
+      expect(page).to have_css("img[src*='#{@item_1.image}']")
```
Comment (adumortier) [inline spec/features/users/merchant_order_show_spec.rb:111]: glad you caught that

**Review by adumortier — APPROVED** [review]

**Comment by TravisBorgsmiller** [comment]
tests and routes look good, way to knock out this extension!

**Review by TravisBorgsmiller — APPROVED** [review]

**Review by adumortier — APPROVED** [review]

**Pull request merged**